global_variables_checker.py

```diff
@@ -24,6 +26,16 @@ def visit_nonlocal(self, node):
         args = "you used the keyword 'nonlocal' on line {}".format(node.lineno)
         self.add_message('forbidden-global-variables', node=node, args=args)
 
+    def visit_assign(self, node):
+        if isinstance(node.frame(), astroid.scoped_nodes.Module):
+            regex = str(node.targets[0])
+            s = re.findall('\((.*?)\)', regex)[0]
+            a = re.match('(([A-Z_][A-Z0-9_]*)|(__.*__))$', s)
```

Comment:
You likely weren't importing the correct thing. Try doing `from pylint.checkers.base import CONST_NAME_RGX`.